spec/client_spec.rb

```diff
@@ -133,7 +133,7 @@
       subject { client.index_documents(engine_name, documents) }
 
       it 'should return an array of document status hashes' do
-        expect(subject).to match(
+        expect(subject.json).to match(
```

Comment:
The first thing popped in my head when I saw this ... is what if someone was relying on the response to be of a particular type?

I.e., is it possible that someone would have had logic in place that checked to see if the response was a Hash or an Array? This would break that check, no?